ciri/usage.h

```diff
@@ -185,7 +185,7 @@ static struct cli_argument_s {
     {"local-snapshots-min-depth", CONF_LOCAL_SNAPSHOTS_MIN_DEPTH,
      "Minimal milestones depth for new local snapshot entry point.", REQUIRED_ARG},
     {"local-snapshots-path-base", CONF_LOCAL_SNAPSHOTS_PATH_BASE,
```

Comment:
Maybe we should also change the option name then ?

Reply:
done